js/src/product-feed/issues-table-card/index.js

```diff
@@ -2,63 +2,25 @@
  * External dependencies
```

Comment:
Table, headers and pagination,  were refactored and moved to their own components 

Now this just check if there are issues and if any, it just renders the <IssuesTypeNavigation /> and the <IssuesTable />